pkg/analysis/analysis.go

```diff
@@ -35,6 +35,7 @@ type AnalysisStatus string
 const (
 	StateOK              AnalysisStatus = "OK"
 	StateProblemDetected AnalysisStatus = "ProblemDetected"
+	MaxConcurrent        int            = 10
```

Comment:
Nice! Shouldn't we add a flag `--concurrency` to the analyzer so that we can set the maximum concurrent value?

Reply:
Yep leave it with me